src/ykman.ts

```diff
@@ -0,0 +1,2354 @@
+                  isRequired: true,
+                  args: {
+                    name: "TEXT",
+                  },
+                },
+                {
+                  name: ["-d", "--valid-days"],
+                  description: "Number of days until the certificate expires",
+                  args: {
+                    name: "INTEGER",
+                    default: "365",
+                  },
+                },
+                {
+                  name: ["-a", "--hash-algorithm"],
+                  description: "Hash algorithm",
+                  args: {
+                    name: "SHA1|SHA256|SHA384|SHA512",
+                    default: "SHA256",
+                  },
+                },
+                {
+                  name: ["-h", "--help"],
+                  description:
+                    "Show piv certificates generate usage information",
+                },
+              ],
+            },
+            {
+              name: "import",
+              description: "Import an X.509 certificate",
+              args: [
+                {
+                  name: "SLOT",
+                  description: "PIV slot of the certificate",
+                },
+                {
+                  name: "CERTIFICATE",
+                  description:
+                    "File containing the certificate. Use '-' to use stdin",
+                },
+              ],
+              options: [
+                {
+                  name: ["-m", "--management-key"],
+                  description: "The management key",
+                  args: {
+                    name: "TEXT",
+                  },
+                },
+                {
+                  name: ["-P", "--pin"],
+                  description: "PIN code",
+                  args: {
+                    name: "TEXT",
+                  },
+                },
+                {
+                  name: ["-p", "--password"],
+                  description: "A password may be needed to decrypt the data",
+                  args: {
+                    name: "TEXT",
+                  },
+                },
+                {
+                  name: ["-v", "--verify"],
+                  description:
+                    "Verify that the certificate matches the private key in the slot",
+                },
+                {
+                  name: ["-h", "--help"],
+                  description: "Show piv certificates import usage information",
+                },
+              ],
+            },
+            {
+              name: "request",
+              description: "Generate a Certificate Signing Request (CSR)",
+              args: [
+                {
+                  name: "SLOT",
+                  description: "PIV slot of the certificate",
+                },
+                {
+                  name: "PUBLIC-KEY",
+                  description:
+                    "File containing a public key. Use '-' to use stdin",
+                },
```

Comment:
Same thing for the arg below this one, can have a template and suggestions.
```suggestion
                {
                  name: "PUBLIC-KEY",
                  description:
                    "File containing a public key. Use '-' to use stdin",
                  template: "filepaths",
                  suggestions: [{ name: "-", hidden: true }],
                },
```

Reply:
Fixed in ce1664646dac9df1dd1d72f8c0130935bfe66b38